{"name": "test_loginWithExcel[case1]", "status": "passed", "parameters": [{"name": "case", "value": "{'id': 2, 'url': 'http://192.168.31.70:5000/login', 'body': '{\"username\": \"user1\", \"password\": \"password\"}', 'desc': '密码错误', 'method': 'post', 'expected': '{\"message\": \"用户名或密码错误\"}'}"}], "start": 1693149051229, "stop": 1693149051232, "uuid": "6eea2393-4aac-4089-9efd-4544a9f21a40", "historyId": "ba0af5b9f87235769a9ca7037dd72280", "testCaseId": "eb7dfc31d29c38e11d64cf1bfcf7f2a3", "fullName": "testcase.test_sever#test_loginWithExcel", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_sever"}, {"name": "host", "value": "PC"}, {"name": "thread", "value": "20640-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_sever"}]}